### PR TITLE
Split multiple resource yaml files

### PR DESCRIPTION
**Pull request by zxDiscovery**
Helm chart recommend the best way is one yaml file only include one resource. So split multiple resource in one yaml file to multiple yaml files with one resource.

### CODE REVIEW

**Comment by zxDiscovery** [comment]
I signed the CLA

**Comment by ostromart** [comment]
@mandarjog could you verify this change is a good idea and doesn't disrupt anything? I believe you were originally involved in creating the resource groupings.

**Comment by gyliu513** [comment]
@zxDiscovery please make sure you singed CLA.

File: install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml]: @zxDiscovery @morvencao can you help check if we can the folder `cleanup` under `templates`? Does this works? I found most of the charts only including files under `templates`, but no folder.

**Comment by zxDiscovery** [comment]
@googlebot  I signed it!

File: install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml (hunk 1 above)
Comment (morvencao) [reply install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml]: @gyliu513 Local tests show we can put some templates into a folder like `cleanup`.

**Comment by zxDiscovery** [comment]
I signed it!

**Comment by zxDiscovery** [comment]
I signed it!

**Comment by zxDiscovery** [comment]
The CLA can't be signed,  I need to resubmit this PR. So close this one

File: install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml (hunk 1 above)
Comment (gyliu513) [reply install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml]: Good to know, thanks @morvencao

**Comment by zxDiscovery** [comment]
New issue at here https://github.com/istio/istio/pull/8864

**Pull request closed without merging**